packages/react-components/react-skeleton/library/src/components/Skeleton/useSkeletonStyles.styles.ts

```diff
@@ -1,17 +1,167 @@
-import { mergeClasses } from '@griffel/react';
+import { makeStyles, mergeClasses } from '@griffel/react';
 import type { SkeletonSlots, SkeletonState } from './Skeleton.types';
 import type { SlotClassNames } from '@fluentui/react-utilities';
+import { tokens } from '@fluentui/react-theme';
```

Comment:
Why do you need to change styles? This feature requires only to pass `size` and `shape` props. You'll need to add context for this